### DISCUSSION TITLE
Configuration of Contention free Random Access(CFRA) for handover

### DISCUSSION THREAD

**Discussion by laxmandvb** [srsran/srsRAN_Project#775] [General Help] [answered]
Hi,
We are trying Inter DU handover. But we are not sure the handover is based on CFRA because we are unable to find CFRA IE in the rrcreconfiguration message. 
![Inter_DU_Handover](https://github.com/user-attachments/assets/0d523f14-835c-49ad-b29d-0492f25395e7)

Kindly  suggest how to configure and verify CFRA procedure for handover testing.

**Comment by herlesupreeth** [comment, marked as answer]
Contention free Random Access is not supported in srsRAN_Project

**Reply by laxmandvb** [reply]
Thanks for the reply.

But how inter DU handover is happening without rach-ConfigDedicated IE (cfra)?

Kindly give more details about inter DU handover.

Regards

**Reply by herlesupreeth** [reply]
I dont think Contention free Random Access is a prerequisite for inter DU handover. But maybe @frankist can confirm
